src/components/CurrencyInputPanel/CurrencyInputPanel.component.tsx

```diff
@@ -145,27 +146,29 @@ export const CurrencyInputPanelComponent = ({
                   data-testid={'transaction-value-input'}
                 />
               </>
-            )}
-            <CurrencySelect
-              selected={!!(currency || pair)}
-              className="open-currency-select-button"
-              disableCurrencySelect={disableCurrencySelect}
-              onClick={handleCurrencySelectClick}
-              disabled={(isLoading || disableCurrencySelect) && !currency}
-            >
-              <Aligner>
-                {isLoading ? (
-                  <Loader style={{ width: '97.25px' }} />
-                ) : (
-                  <CurrencyView
-                    pair={pair}
-                    currency={currency}
-                    chainIdOverride={chainIdOverride}
-                    currencyWrapperSource={currencyWrapperSource}
-                  />
-                )}
-              </Aligner>
-            </CurrencySelect>
+            )}{' '}
+            <CurrencySelectWrapper currency={currency}>
```

Comment:
yeah from the code I can't see why its passed as prop as opposed to just adding the wrapper?

Reply:
We can't show this tooltip in Swap and Bridge boxes, so it's being passed as a prop.

Reply:
Took a look into it and I find this the best name. It's a generic name, so if we want to add a different component wrapping `CurrencySelect` we can do it, it's not tied to a tooltip. The name tooltip is being used in the `CurrencySelectTooltip` already, and here makes sense cause it's a tooltip.